### PR TITLE
Add deinterlace on VA module.

### PR DESCRIPTION
**Pull request by feiwan1**
Jira: None.
Test: Build passes on Linux and Android.

Signed-off-by: Wang,Fei <fei.w.wang@intel.com>

### CODE REVIEW

**Comment by feiwan1** [comment]
@kalyankondapally @XiaosongWei , please have a review. Thanks.

**Comment by kalyankondapally** [comment]
I will wait for @XiaosongWei  to review.

File: common/compositor/va/varenderer.cpp [hunk 1]
```diff
@@ -149,6 +157,49 @@ bool VARenderer::SetVAProcFilterColorValue(HWCColorControl mode,
...
+  if (prop.flag == HWCDeinterlaceFlag::kDeinterlaceFlagNone) {
+    SetVAProcFilterDeinterlaceDefaultMode();
+    return true;
+  } else if (prop.flag == HWCDeinterlaceFlag::kDeinterlaceFlagForce ||
+             GetVAProcDeinterlaceFlagFromVideo(prop.flag)) {
+    if (prop.mode == HWCDeinterlaceControl::kDeinterlaceNone) {
+      deinterlace_caps_.mode_ = VAProcDeinterlacingNone;
+    } else if (prop.mode == HWCDeinterlaceControl::kDeinterlaceBob) {
+      deinterlace_caps_.mode_ = VAProcDeinterlacingBob;
+    } else if (prop.mode == HWCDeinterlaceControl::kDeinterlaceWeave) {
```
Comment (XiaosongWei) [inline common/compositor/va/varenderer.cpp]: using switch instead of if-else?

Comment (feiwan1) [reply common/compositor/va/varenderer.cpp]: @XiaosongWei , updated.

**Comment by XiaosongWei** [comment]
lgtm

**Pull request closed without merging**